rotkehlchen/tests/unit/decoders/test_aave.py

```diff
@@ -216,3 +223,668 @@ def test_aave_eth_withdraw_v1(database, ethereum_inquirer):
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xde6857219544bb5b7746f48ed30be6386fefc61b2f864cacf559893bf50fd951'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000002715273613632226985186221669179813245119'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000000000000000000000000000000000000000000000'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbevmtx = DBEvmTx(database)
+    with dbevmtx.db.user_write() as cursor:
+        dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+    decoder = EthereumTransactionDecoder(
+        database=database,
+        ethereum_inquirer=ethereum_inquirer,
+        transactions=eth_transactions,
+    )
+    with dbevmtx.db.user_write() as cursor:
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO, usd_value=ZERO),
+            location_label='0x2715273613632226985186221669179813245119',
+            notes='Burned 0 ETH for gas',
+            counterparty=CPT_GAS,
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=419,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_WETH,
+            balance=Balance(amount=FVal(1500)),
+            location_label='0x2715273613632226985186221669179813245119',
+            notes='Deposit 1500 eip155:1/erc20:0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 into AAVE V2',  # noqa: E501
+            counterparty='aave-v2',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=420,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=EvmToken('eip155:1/erc20:0x030bA81f1c18d280636F32af80b9AAd02Cf0854e'),  # aWETH
+            balance=Balance(amount=FVal(1500)),
+            location_label='0x2715273613632226985186221669179813245119',
+            notes='Receive 1500 aWETH from 0x0000000000000000000000000000000000000000 to 0x2715273613632226985186221669179813245119',  # noqa: E501
```

Comment:
This note is strange... In maker we say something similar to `Generate 1500 aWETH by {address}`